molgenis/molgenis-emx2
# chore: add tools for typescript linting

What are the main changes you did:
- add dependencies
- add script to package.json

run with `yarn typecheck`

how to test:
check out the code 
install the dependancies ( via gradle or yarn )
run the tool, from the app folder; yarn typecheck
see output 

todo:
